libs/klv/src/main/java/org/codice/alliance/libs/klv/DistinctKlvProcessor.java

```diff
@@ -41,7 +41,7 @@ protected void doProcess(Attribute attribute, Metacard metacard) {
         attribute
             .getValues()
             .stream()
-            .filter(Utilities::isNotEmptyString)
+            .filter(this::isValidAttributeValue)
```

Comment:
Why is this function needed over Utilities::isNotBlankString

Reply:
It gives me a hook to override what's defined as a valid attribute value in the `AlphanumericDistinctKlvProcessor`.